personal_trainer/demo/account_analytic_line.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="account_analytic_line_1" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date">2024-09-16</field>
+        <field name="project_id" ref="project_project_3"/>
+        <field name="task_id" ref="project_task_18"/>
```

Comment:
This sale order line generates a project, not a task. Not sure it works as expected